data/saas/config/snap_config.yml

```diff
@@ -46,6 +46,9 @@ saas_config:
               value: snapchat-marketing-api
         token_request:
           method: POST
+          client_config:
+            protocol: https
+            host: accounts.snapchat.com
```

Comment:
The `token_request` and `refresh_request` both needed to use a different `host`, accounts.snapchat.com